tools/topology/m4/muxdemux.m4

```diff
@@ -2,6 +2,24 @@ divert(-1)
 
 dnl Define macro for demux widget
 
+define(comp_type, 0)
```

Comment:
what is 0?

Reply:
this is not used for anything and will be removed in #2600, so I will remove it also